TFT/src/User/API/config.c

```diff
@@ -811,6 +811,10 @@ void parseConfigKey(uint16_t index)
       infoSettings.xy_offset_probing = getOnOff();
       break;
 
+    case C_INDEX_Z_RAISE_PROBING:
+      SET_VALID_FLOAT_VALUE(infoSettings.z_raise_probing, MIN_Z_POS_LIMIT, MAX_SIZE_LIMIT);
```

Comment:
Using `MIN_Z_POS_LIMIT` as the minimum will prevent Delta's from being able to set it to a negative value, since `MIN_Z_POS_LIMIT` is currently set to 0.

Reply:
thanks for the feedback. PR  #1718 fixes the XYZ min limit to -2000 to support also Delta printers